src/Commands.rb

```diff
@@ -2095,6 +2095,34 @@ def event_look_at?(ev, to, scope, metric = :square)
       end
       return x_axis && y_axis && (distance_between(metric, ev, to)<=scope)
     end
+    def event_look_towards_event?(ev1, ev2, scope)
```

Comment:
### [1] Parameters' naming

Using `ev1` and `ev2` is not very legible 😓.
At least, would it be possible for you to change `ev1` into something like `observer_ev` ?

Reply:
observer_ev seems to be a bit too verbose.
I'll think about something.